packages/core/lib/commands/migrate/meta.js

```diff
@@ -62,7 +66,7 @@ module.exports = {
       "                                " + // spacing to align with previous line
       "[--compile-all] [--compile-none] [--verbose-rpc] [--interactive]\n" +
       "                                " + // spacing to align with previous line
-      "[--skip-dry-run] [--describe-json] [--dry-run]",
+      "[--skip-dry-run] [--describe-json] [--dry-run] [--network <network>|--url <provider_url>]",
```

Comment:
Can we add the spacing to align with the previous line?

Reply:
Sure @lsqproduction. I will add the spacing.